### PR TITLE
Logger gs

### PR DESCRIPTION
**Pull request by ariamirzai**
- Need to test if the "ControlTower Logs" folder is created inside the "Documents"-directory on Windows.
- The car outputted in RCCar when starting the application can look weird once sent over to ControlTower, I suggest only printing it to the RCCar-terminal through e.g. printf(), instead of using qDebug().

Possible improvements to the STATUSTEXT transfers:
- Implement message-acknowledgements to make sure all messages arrive.
- Use the "chunk_seq" field provided by STATUSTEXT to make sure chunks of messages are always re-assembled in the correct order using a sorting function.

### CODE REVIEW

**Comment by ariamirzai** [comment]
main.cpp and mainwindow.cpp have been updated for compatability the the revised Logger class.

File: mainwindow.cpp [hunk 1]
```diff
@@ -14,12 +15,16 @@ MainWindow::MainWindow(QWidget *parent)
...
+
+    connect(&Logger::getInstance(), &Logger::logSent, this, &MainWindow::on_logSent);
+
     ui->mapWidget->setScaleFactor(0.05);
     ui->mapWidget->setSelectedObjectState(0);
     ui->mapWidget->addMapModule(ui->planUI->getRoutePlannerModule());
     ui->mapWidget->addMapModule(ui->traceUI->getTraceModule());
     ui->mapWidget->addMapModule(ui->flyUI->getGotoClickOnMapModule());
     ui->mapWidget->addMapModule(ui->cameraGimbalUI->getSetRoiByClickOnMapModule());
+    ui->logBrowser->hide();
```
Comment (marvind) [inline mainwindow.cpp:27]:
```suggestion

    Logger::initGroundStation();

    ui->setupUi(this);
    ui->logBrowser->hide();

    connect(&Logger::getInstance(), &Logger::logSent, this, &MainWindow::on_logSent);

    ui->mapWidget->setScaleFactor(0.05);
    ui->mapWidget->setSelectedObjectState(0);
    ui->mapWidget->addMapModule(ui->planUI->getRoutePlannerModule());
    ui->mapWidget->addMapModule(ui->traceUI->getTraceModule());
    ui->mapWidget->addMapModule(ui->flyUI->getGotoClickOnMapModule());
    ui->mapWidget->addMapModule(ui->cameraGimbalUI->getSetRoiByClickOnMapModule());
```

Comment (marvind) [reply mainwindow.cpp:27]: Would suggest to execute Logger::initGroundStation(); here, move ui->logBrowser->hide(); to the setupUI call

**Review by marvind — APPROVED** [review]
Minor comments. Well done and ready to merge!

File: mainwindow.cpp (hunk 1 above)
Comment (ariamirzai) [reply mainwindow.cpp]: Suggested changes has now been implemented

**Comment by rickaha** [comment]
Don't have any additional comments on the syntax, but i haven't tested that everything works

**Pull request merged**